packages/api-cli/src/api.ts

```diff
@@ -163,14 +170,17 @@ async function getCallInfo (): Promise<CallInfo> {
   return {
     api,
     fn,
-    log: (result: SubmittableResult | Codec | ApiCallFn): void => console.log(
-      JSON.stringify({
-        // eslint-disable-next-line @typescript-eslint/unbound-method
-        [method]: isFunction((result as Codec).toHuman)
-          ? (result as Codec).toHuman()
-          : result
-      }, null, 2)
-    ),
+    log: (result: SubmittableResult | Codec | ApiCallFn): void =>
+      console.log(
+        JSON.stringify(
+          {
+            // eslint-disable-next-line @typescript-eslint/unbound-method
+            [method]: isFunction((result as Codec).toHuman) ? (result as Codec).toHuman() : result
+          },
+          null,
+          2
+        )
```

Comment:
Formatting here is quite weird, not sure why.

Reply:
Me neither. The linter doesn't complain